solc/CommandLineInterface.cpp

```diff
@@ -154,6 +162,14 @@ static set<string> const g_combinedJsonArgs{
 	g_strSrcMapRuntime
 };
 
+/// Possible arguments to for --machine
+static set<string> const g_machineArgs
+{
+	g_strEVM,
+	g_strEVM15,
+	g_streWASM
```

Comment:
Perhaps could go with `eWasm` to be in line with the rest (or change the rest to `eWASM`).